src/ucp/stream/stream_send.c

```diff
@@ -34,6 +34,7 @@ static void ucp_stream_send_req_init(ucp_request_t* req, ucp_ep_h ep,
     req->send.datatype     = datatype;
     req->send.mem_type     = UCT_MD_MEM_TYPE_HOST;
     req->send.lane         = ep->am_lane;
+    req->send.type         = UCP_TAG_TYPE;
```

Comment:
It looks confusing... pls add STREAM_TYPE. Also I would add more clear prefix, e.g. UCP_REQUEST_TYPE_SEND_<AM|TAG|STREAM>